packages/utils/src/utils/index.js

```diff
@@ -32,11 +32,17 @@ export function getErrorMessage(error) {
 }
 
 export function selectedTask(selectedTaskName, tasks) {
-  return tasks.find(t => t.metadata.name === selectedTaskName);
+  if (tasks) {
+    return tasks.find(t => t.metadata.name === selectedTaskName);
+  }
+  return {};
 }
 
 export function selectedTaskRun(selectedTaskId, taskRuns = []) {
-  return taskRuns.find(run => run.id === selectedTaskId);
+  if (taskRuns) {
```

Comment:
taskRuns should always be defined (we default to `[]`) here unless we're somehow passing `null` instead of `undefined`

if that's the case it should be caught before we get here
similar for `selectedTask` above, ideally we wouldn't get that far without tasks